framework-test/src/main/java/org/checkerframework/framework/test/diagnostics/TestDiagnosticUtils.java

```diff
@@ -356,6 +356,11 @@ public static TestDiagnosticLine fromJavaSourceLine(
           fromJSpecifyFileComment(filename, errorLine, trimmedLine.substring(3));
       return new TestDiagnosticLine(
           filename, errorLine, line, Collections.singletonList(diagnostic));
+    } else if (trimmedLine.startsWith("// test:cannot-convert")) {
```

Comment:
You want to do it this way rather than using a regexp like `^// (jspecify_|test:cannot-convert)`? Otherwise you'll end up copying a bunch of lines a bunch of times.

Reply:
Good point. I figure that `||` is simplest for now, but we should revisit if we end up introducing more conformance-test comments into the `samples` files in place.